### PR TITLE
MINOR Black duck fixes for flyway

### PR DESCRIPTION
**Pull request by raju-saravanan**
* Tested for registry using all the available options in the bootstrap-storage script.
* Tested on a database where migration was done by previous flyway version (4.12) and applied "migrate" option again on the same database with new flyway version (5.1.4)
* These changes have to be tested for streamline and SMM.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -597,6 +598,18 @@
...
+                        <groupId>org.codehaus.plexus</groupId>
+                        <artifactId>plexus-utils</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+
+            <dependency>
+                <groupId>org.codehaus.plexus</groupId>
+                <artifactId>plexus-utils</artifactId>
+                <version>3.1.0</version>
```
Comment (priyank5485) [inline pom.xml]: Use the version property above.

File: storage/tool/pom.xml [hunk 2]
```diff
@@ -44,10 +44,19 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-simple</artifactId>
         </dependency>
+        <dependency>
```
Comment (priyank5485) [inline storage/tool/pom.xml]: Any reason for adding commons-io ?

**Comment by satishd** [comment]
@arunmahadevan @HeartSaVioR Can you test/verify these changes on streamline and put your comments?

File: storage/tool/pom.xml (hunk 2 above)
Comment (raju-saravanan) [reply storage/tool/pom.xml]: I am using it ShellMigrationResolver.java where we build checksum of the migration from the content of the file read as a byte array. To read the content as byte array I am using IOUtils from commons-io.

**Comment by raju-saravanan** [comment]
@priyank5485 : Thanks for testing this, once @arunmahadevan  and @HeartSaVioR review changes and test it against SAM we can merge this PR.

**Pull request closed without merging**